GymEventManager/Controllers/Event Details View/EventDetailsViewController.swift

```diff
@@ -8,37 +8,53 @@
 import UIKit
 
 class EventDetailsViewController: UIViewController {
-
-    var event: Event?
+    
+    // MARK: - IBOutlet Variables
     @IBOutlet weak var eventNameLabel: UIButton!
     @IBOutlet weak var tableView: UITableView!
+    
+    // MARK: - Properties
+    var event: Event?
+
+    // MARK: - LifeCycle Functions
     override func viewDidLoad() {
-        super.viewDidLoad()
+        setupView()
+    }
+   
+    private func setupView() {
+        setupTableView()
+        setupEventNameLabel()
+    }
+    
+    private func setupEventNameLabel() {
+        if let currentEvent = event {
+            eventNameLabel.titleLabel?.text?.append(currentEvent.name)
+        }
+    }
+    
+    private func setupTableView() {
```

Comment:
try to use descriptive name as much as you can not TableViewCell and header cell